kubernetes/istio-15.yaml

```diff
@@ -232,17 +232,17 @@ spec:
 apiVersion: v1
 kind: Service
 metadata:
-  name: istio-ingress-controller
+  name: istio-ingress
   labels:
-    infra: istio-ingress-controller
+    infra: istio-ingress
```

Comment:
Does it make sense to unify the label key here with other Istio components, e.g. ""istio: ingress", "istio: manager", "istio: mixer"?